locales/ja-JP.yml

```
@@ -828,6 +828,15 @@ reloadToApplySetting: "設定はページリロード後に反映されます。
 needReloadToApply: "反映には再起動が必要です。"
 showTitlebar: "タイトルバーを表示する"
 clearCache: "キャッシュをクリア"
+timelineCache: "タイムラインのキャッシュ"
+timelineCacheDescription: "自分のタイムライン、リスト、アンテナのキャッシュを削除します。"
+purgeHomeTimelineCache: "ホームタイムラインのキャッシュを削除"
+purgeUserTimelineCache: "ユーザータイムラインのキャッシュを削除"
+selectUserList: "リストを選択"
+purgeUserListTimelineCache: "リストタイムラインのキャッシュを削除"
+selectAntenna: "アンテナを選択"
```

YAMLlint (1.37.1)

[error] duplication of key "selectAntenna" in mapping

(key-duplicates)